<ui version="4.0" >
 <class>Settings</class>
 <widget class="QDialog" name="Settings" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>169</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="3" >
    <layout class="QHBoxLayout" >
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="label" >
           <property name="text" >
            <string>Console Scrollback:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="ConsoleScrollbackBox" >
           <property name="suffix" >
            <string> lines</string>
           </property>
           <property name="minimum" >
            <number>1</number>
           </property>
           <property name="maximum" >
            <number>1000</number>
           </property>
           <property name="singleStep" >
            <number>1</number>
           </property>
           <property name="value" >
            <number>100</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="label_3" >
           <property name="text" >
            <string>Output Scrollback:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="OutputScrollbackBox" >
           <property name="wrapping" >
            <bool>false</bool>
           </property>
           <property name="suffix" >
            <string> lines</string>
           </property>
           <property name="minimum" >
            <number>1</number>
           </property>
           <property name="maximum" >
            <number>1000</number>
           </property>
           <property name="singleStep" >
            <number>10</number>
           </property>
           <property name="value" >
            <number>100</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="label_2" >
           <property name="text" >
            <string>Device</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="DeviceEdit" >
           <property name="text" >
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>Make Device empty to autodetect</string>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2" >
       <property name="title" >
        <string>Tree</string>
       </property>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QCheckBox" name="ShowTypeBox" >
          <property name="text" >
           <string>Show Type</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ShowMemBox" >
          <property name="text" >
           <string>Show Mem</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ShowClearBox" >
          <property name="text" >
           <string>Show Clear</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ShowPermissionsBox" >
          <property name="text" >
           <string>Show Permissions</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>91</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" >
    <widget class="QPushButton" name="SaveButton" >
     <property name="text" >
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QPushButton" name="CloseButton" >
     <property name="text" >
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>CloseButton</sender>
   <signal>clicked()</signal>
   <receiver>Settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>389</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel" >
     <x>153</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
